packages/widgetbook/lib/src/fields/color_field/color_text_field.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+class ColorTextField extends StatelessWidget {
```

Comment:
This can be refactored into `NumberTextField` as it's always used for numbers.
Then it can also be used for the alpha percentage.